src/Controls/src/Core/Handlers/Shapes/BoxView/BoxViewHandler.Windows.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Maui.Controls.Handlers
+{
+	public partial class BoxViewHandler : ShapeViewHandler
+	{
+		public override bool NeedsContainer 
+			=> false;
```

Comment:
Is this not true for all platforms? Since boxview is not really a thing, this should just work with the view?

Reply:
I don't think I fully understand what Matt's asking, but I _think_ the idea here is that since Windows supports Brushes natively, BoxView doesn't need to add a wrapper for dealing with that sort of thing on the Windows platform.

Reply:
Right, is not necessary and also is wrapping the platform BoxView with the WrapperView. Then is not handled correctly and because the platform BoxView already has Parent, crashes under some conditions.